ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -58,20 +62,14 @@ public static boolean shouldScrub(Map<String, String> attributes, JSONObject jso
         && (attributes.get(Attribute.APP_VERSION).startsWith("68")
             || attributes.get(Attribute.APP_VERSION).startsWith("69"))
         && Optional.of(json) // payload.hangs[].remoteType
-            .map(j -> j.optJSONObject("payload")) //
-            .map(j -> {
-              Boolean isPresent = false;
-              JSONArray arr = j.optJSONArray("hangs");
-              for (int i = 0; i < arr.length(); i++) {
-                String s = arr.optJSONObject(i).optString("remoteType");
-                if (s != null && s.startsWith("webIsolated=")) {
-                  isPresent = true;
-                  break;
-                }
-              }
-              return isPresent;
-            }) //
-            .filter(b -> b).isPresent()) {
+            .map(j -> j.get("payload")) //
+            .map(j -> j.get("hangs")) //
+            .filter(JsonNode::isArray).map(ArrayNode.class::cast) //
+            .map(ArrayNode::elements) //
+            .map(Streams::stream).orElseGet(Stream::empty) //
```

Comment:
`Streams.stream(json.path("payload").path("hangs").elements())` should work even if `payload.hangs` isn't an array, or isn't present